thymleafdemo/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+#
+# JDBC properties
+#
+spring.datasource.url=jdbc:mysql://localhost:3306/employee_directory
+spring.datasource.username=springstudent
+spring.datasource.password=springstudent
```

Comment:
@avantibonde try using env based properties file. As in real-world dev and prod DB can be different